lib/flutter_assurance.dart

```diff
@@ -3,19 +3,28 @@ Copyright 2020 Adobe. All rights reserved.
 This file is licensed to you under the Apache License, Version 2.0 (the "License");
 you may not use this file except in compliance with the License. You may obtain a copy
 of the License at http://www.apache.org/licenses/LICENSE-2.0
-
 Unless required by applicable law or agreed to in writing, software distributed under
 the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
 OF ANY KIND, either express or implied. See the License for the specific language
 governing permissions and limitations under the License.
 */
 
-#import <ACPGriffon/ACPGriffonEvent.h>
```

Comment:
Confused about this file It was .h and u change it to .dart ?

Reply:
Nope I have deleted the ACPGriffonEvent+Flutter.h file 
Git got confused with the dart file because of the header I guess.

Reply:
Looks all good